app/layout.tsx

```diff
@@ -0,0 +1,54 @@
+import './globals.css';
+import { Source_Sans_3, Source_Serif_4 } from 'next/font/google';
+import { storyblokInit, apiPlugin } from '@storyblok/react/rsc';
+import StoryblokProvider from '@/components/StoryblokProvider';
```

Comment:
This is the new layout.tsx file for the app directory approach. Every page file in the app directory makes use of this common layout. Font loading is done here as well. In #97 I will also deal with local fonts using the nextjs approach (I've refactored the Google fonts here)